parallel-install/pkg/engine/engine.go

```diff
@@ -1,3 +1,10 @@
+//Package engine defines the contract and implements parallel processing of components.
+//The Engine is configured with a number of workers that run in parallel.
+//If only single worker is configured, the processing becomes sequential.
+//If you need different configuration for installation and uninstallation,
```

Comment:
User is not able to use his own instance of Engine to start installation/uninstallation. He can use different Installation, though.

Reply:
Because the type is public I documented it from the type's perspective, not how it's used in top-level API. Leaving as it is.